app/components/course-page/header/navigation-controls.ts

```diff
@@ -38,6 +42,17 @@ export default class NavigationControls extends Component<Signature> {
   get currentUser() {
     return this.authenticator.currentUser;
   }
+
+  @action
+  handleCloseButtonClick() {
+    if (this.args.track) {
+      this.router.transitionTo('track', this.args.track);
+    } else if (this.args.activeRepository && this.args.activeRepository.firstSubmissionCreated) {
```

Comment:
```suggestion
    } else if (this.args.activeRepository && this.args.activeRepository.language) {
```

(We don't need to require that a user pushed a submission here, even if they just started the course it's fair to take them to the track page)